whois-update/src/main/java/net/ripe/db/whois/update/keycert/PgpSignedMessage.java

```diff
@@ -176,7 +177,7 @@ public boolean verify(final PGPPublicKey publicKey) {
 
     // The signing time must be within an hour of the current time.
     public boolean verifySigningTime(final DateTimeProvider dateTimeProvider) {
-        final LocalDateTime signingTime = DateTimeProvider.fromDate(getPgpSignature().getCreationTime());
+        final LocalDateTime signingTime = DateUtil.fromDate(getPgpSignature().getCreationTime());
```

Comment:
Tested ok (sent signed message via syncupdates and mailupdates, updated object ok)